docs/BR.md

```diff
@@ -122,6 +122,8 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
 | 1.6.7 | SC23 | Precertificates | 14-Nov-2019	| 19-Dec-2019 |
 | 1.6.7 | SC24 | Fall Cleanup v2 | 12-Nov-2019	| 19-Dec-2019 |
 | 1.6.8 | SC25 | Define New HTTP Domain Validation Methods v2 | 31-Jan-2020	| 3-Mar-2020 |
+| 1.6.9 | SC27 | Version 3 Onion Certificates v3 | 19-Feb-2020	| 27-Mar-2020 |
```

Comment:
This should be "Version 3 Onion Certificates" instead of "Version 3 Onion Certificates v3"

Reply:
I removed "v3" from the version table, but I will note that the version number of other ballots has been included in that table